string-handling/string-characteristics/c-sharp/change_string.cs

```diff
@@ -0,0 +1,45 @@
+/*
+Isaac Computer Science
+Usage licensed under the Open Government Licence v3.0
+Note: This file is designed to be copied out and compiled on your machine.
+In order for it to compile properly you need to ensure that the project name is the same as the "namespace" in this file.
+To run this file you need to:
+1. Copy the contents
+2. Paste them into the C# IDE of your choice (Visual Studio, for example)
+3. Change the namespace to match your project (if neccesary)
+4. Compile the program
+5. Run the program
+*/
+
+using System;
+
+namespace IsaacCodeSamples
+{
+
+    class StringCharacteristics
+    {
+
+        // The Main method is the entry point for all C# programs
+        public static void Main() {
+            string myString = "An aardvark is an animal";
+            string newString = ChangeString(myString);
+            Console.WriteLine(newString);
+        }
+
+
+        // Demonstrates how to use ASCII codes to convert letter case
+        public static string ChangeString(string myString) {
+            string newString = "";  // New empty string
+            foreach (var character in myString) {
+                if (character == Convert.ToChar("a")) { 
```

Comment:
I'm assuming that the iterator gets a unicode value and that's why you need to convert "a" to a unicode value here?

Reply:
Yes - this is explained above the code snippet.

Reply:
Why does `var character` get the unicode value of the character and not (for example) `'a'`? i.e. why isn't it

```
foreach( char character in myString){
    if(character == "a"){
        ...etc
```

?